common/src/main/java/org/apache/celeborn/common/network/client/TransportResponseHandler.java

```diff
@@ -168,12 +190,22 @@ public void handle(ResponseMessage message) throws Exception {
       RpcResponse resp = (RpcResponse) message;
       RpcResponseCallback listener = outstandingRpcs.get(resp.requestId);
       if (listener == null) {
-        logger.warn(
-            "Ignoring response for RPC {} from {} ({} bytes) since it is not outstanding",
-            resp.requestId,
-            NettyUtils.getRemoteAddress(channel),
-            resp.body().size());
-        resp.body().release();
+        listener = outstandingPushs.get(resp.requestId);
```

Comment:
IMO we should check outstandingPushes first before outstandingRpcs, since pushData requests are much more than rpc requests.

Reply:
Done